azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/AbstractFunctionMojo.java

```diff
@@ -19,25 +19,25 @@
 
 public abstract class AbstractFunctionMojo extends AbstractAppServiceMojo {
 
-    private static final String JDK_VERSION_ERROR = "Your JDK Version is %s ,which doesn't fit azure maven " +
-            "function plugin, for now we only support JDK 8";
+    private static final String JDK_VERSION_ERROR = "Local JDK Version %s does not meet the maven " +
```

Comment:
`maven plugin requirement` > `requirement of the Maven plugin for Azure Functions`